example-runner/src/main/resources/values.yaml

```diff
@@ -21,9 +21,7 @@ service:
 image:
   name: @artifactId@
   base: @dockerfile.base.tag@
-  tag: @revision@-@git.commit.id.abbrev@
   pullPolicy: IfNotPresent
-  codeRelease: @project.parent.version@
```

Comment:
deployment-jvm module  needs the same refactoring to remove codeRelease and tag as was done here in example-runner.